### PR TITLE
Update README.md to add appraisal in gemspec instead of Gemfile

### PR DESCRIPTION
**Pull request by aripollak**
I'm not really sure if this is the right thing to do, so this is more of an RFC. I was just trying to set up appraisal in a gem that was created a year ago and the instructions weren't working.

### CODE REVIEW

**Comment by mike-burns** [comment]
It might be worthwhile to call out why this goes in the gemspec instead of the Gemfile.

File: README.md [hunk 1]
```diff
@@ -19,9 +19,9 @@ without interfering with day-to-day development using Bundler.
 Installation
 ------------
 
-In your Gemfile:
+In your package's .gemspec:
 
-    gem "appraisal"
+   s.add_development_dependency 'appraisal'
```
Comment (odlp) [inline README.md]: Please could this be changed to double quotes, so it remains consistent with the rest of the readme?

**Comment by aripollak** [comment]
@odlp done. Looks like the allow-edits box is already checked too.

**Comment by deivid-rodriguez** [comment]
Why this? I actually prefer the other way around, since for example I'm able to point directly to the master branch in github if I need to.

**Comment by aripollak** [comment]
@deivid-rodriguez on a new gem project, putting appraisal in Gemfile wasn't working.

**Comment by deivid-rodriguez** [comment]
mmmm, can you be more specific? What wasn't working? I did a simple test and it worked fine:

```console
deivid@pantani ~/Code $ bundle gem testgem
Creating gem 'testgem'...
MIT License enabled in config
Code of conduct enabled in config
      create  testgem/Gemfile
      create  testgem/.gitignore
      create  testgem/lib/testgem.rb
      create  testgem/lib/testgem/version.rb
      create  testgem/testgem.gemspec
      create  testgem/Rakefile
      create  testgem/README.md
      create  testgem/bin/console
      create  testgem/bin/setup
      create  testgem/.travis.yml
      create  testgem/.rspec
      create  testgem/spec/spec_helper.rb
      create  testgem/spec/testgem_spec.rb
      create  testgem/LICENSE.txt
      create  testgem/CODE_OF_CONDUCT.md
Initializing git repo in /home/deivid/Code/testgem
deivid@pantani ~/Code $ cd testgem/
deivid@pantani ~/Code/testgem $ sed -i 's/TODO/something/' testgem.gemspec 
deivid@pantani ~/Code/testgem $ echo "gem 'appraisal'" >> Gemfile 
deivid@pantani ~/Code/testgem $ bundle
Fetching gem metadata from https://rubygems.org/.........
Fetching version metadata from https://rubygems.org/.
Resolving dependencies...
Using rake 10.5.0
Using bundler 1.14.6
Using thor 0.19.4
Using diff-lcs 1.3
Using rspec-support 3.5.0
Using testgem 0.1.0 from source at `.`
Using appraisal 2.2.0
Using rspec-core 3.5.4
Using rspec-expectations 3.5.0
Using rspec-mocks 3.5.0
Using rspec 3.5.0
Bundle complete! 5 Gemfile dependencies, 11 gems now installed.
Use `bundle show [gemname]` to see where a bundled gem is installed.
deivid@pantani ~/Code/testgem $ 
```

**Comment by aripollak** [comment]
@deivid-rodriguez when using `bundle gem` with Bundler 1.13.6, the auto-generated Gemfile explicitly includes the comment "Specify your gem's dependencies in testgem.gemspec", and example development dependencies are specified in the gemspec. Even if it works as is (I don't remember what wasn't working anymore), Bundler best practices say to use the gemspec.

**Comment by deivid-rodriguez** [comment]
Yes, I've heard that argument before: "I do it because `bundler` tells me to". But I've never been given an actual reason.

My personal best practice is to specify `add_development_dependeny 'bundler'`, and then manage all development dependencies via `Gemfile`. It works great and I find it more flexible.

**Pull request merged**